api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/components/SolrQuery.java

```diff
@@ -0,0 +1,141 @@
+package edu.cornell.mannlib.vitro.webapp.dynapi.components;
+
+import com.google.common.reflect.TypeToken;
+import com.google.gson.Gson;
```

Comment:
Instead of bringing in a new dependency to parse JSON, should we use fasterxml ObjectMapper?

```
private final ObjectMapper objectMapper = new ObjectMapper();
...
TypeReference type = new TypeReference<Map<String, Object>>(){};
return objectMapper.readValue(query, type);
```

May require some exception handling but may be more appropriate than another dependency.

Reply:
Ok, will do.